packages/teams-js/src/internal/communication.ts

```diff
@@ -88,6 +97,12 @@ export function initializeCommunication(validMessageOrigins: string[] | undefine
     }
   }
 
+  // Extend the window, define the NAA listener and add the NAA bridge
+  tryPolyfillWithNestedAppAuthBridge(Communication.currentWindow, {
```

Comment:
This method is being called in the case of a frameless window; I tested it on Mac application. Similarly, it should be invoked for iOS mobile as well. I was able to successfully call the "createNestedAppAuthBridge()" method, which I believe is unexpected because mobile platforms introduce their own NAA bridge. Is it possible to add filter to avoid unnecessary introduction of bridge for mobile?

Reply:
As discussed on chat, I was having outdated changes. Looks good to me.